client/src/App.js

```diff
@@ -17,29 +17,31 @@ function App() {
   }, []);
 
   return (
-    <Router>
-      <GlobalStyle />
-      <Switch>
-        <Route exact path="/">
-          {loading ? <LoadingScreen /> : <HomePage />}
-        </Route>
-        <Route path="/category/:category">
-          <CategoryPage />
-        </Route>
-        <Route path="/browse">
-          <BrowsePage />
-        </Route>
-        <Route path="/projects/:projectId">
-          <DetailsPage />
-        </Route>
-        <Route path="/favourites">
-          <FavouritesPage />
-        </Route>
-        <Route path="/add">
-          <AddProjectPage />
-        </Route>
-      </Switch>
-    </Router>
+    <div className="box">
+      <Router>
+        <GlobalStyle />
+        <Switch>
+          <Route exact path="/">
+            {loading ? <LoadingScreen /> : <HomePage />}
+          </Route>
+          <Route path="/category/:category">
+            <CategoryPage />
+          </Route>
+          <Route path="/browse">
+            <BrowsePage />
+          </Route>
+          <Route path="/projects/:projectId">
+            <DetailsPage />
+          </Route>
+          <Route path="/favourites">
+            <FavouritesPage />
+          </Route>
+          <Route path="/add">
+            <AddProjectPage />
+          </Route>
```

Comment:
so mabye that helps with orientation, dont include it in the actual components, mabye you have to hide the header when its displayed during loading too
```suggestion
          {loading ? <></> : <Header title={"Craftified"} />
          <Route exact path="/">
            {loading ? <LoadingScreen /> : <HomePage />}
          </Route>
          <Route path="/category/:category">
            <CategoryPage />
          </Route>
          <Route path="/browse">
            <BrowsePage />
          </Route>
          <Route path="/projects/:projectId">
            <DetailsPage />
          </Route>
          <Route path="/favourites">
            <FavouritesPage />
          </Route>
          <Route path="/add">
            <AddProjectPage />
          </Route>
      <Navbar />
```